src/Components/Scales/ScalesPage.module.css

```diff
@@ -22,9 +22,14 @@
     height: 100%;
 }
 
+.scaleSelectorContainer{
+    grid-row-start: 3;
+    width: 100%;
```

Comment:
in .mainContainer, grid-template-rows only defined 2 section (15% and 35%, auto is for empty spaces). 15% is for the note selector and 35% is for the scale, you should add another row to this and assign it to scaleSelectorContainer